docker/CMSRucioClient/scripts/cmslinks.py

```diff
@@ -53,11 +56,20 @@ def _get_rselist(self, rselist=None):
 
         for rse in rselist:
             attrs = self.rcli.list_rse_attributes(rse=rse)
+            pnn = attrs.get('pnn')
+            if pnn is None:
+                project = gl.projects.get('SITECONF/'+rse)
```

Comment:
Hi @amanrique1 
I think `rse` is not just the `real` rse here, but all temp and test, which do not have a separate siteconf.

To test your gitlab functions and operations. You can clone the siteconf repos. One or two is good enough. And use your Personal Access Token on that siteconf clone.